packages/qdrant-loader-core/tests/test_registry.py

```
@@ -0,0 +1,12 @@
+import pytest
+from qdrant_loader_core.graph.registry import EntityExtractor
+
+
+def test_registry_lookup():
+    extractor = EntityExtractor.for_source("jira")
+    assert extractor is not None
+
+
+def test_registry_invalid():
+    with pytest.raises(ValueError):
```

Ruff (0.15.14)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)